Sketchfab-Exporter.pyp

```diff
@@ -245,27 +276,40 @@ class PublishModelThread(threading.Thread):
 
         finally:
             # Clean up
-            #logging.debug("Cleaning up...")
+            # logging.debug("Cleaning up...")
             self.cleanup_files(archiveName, exportFile)
             c4d.SpecialEventAdd(__plugin_id__)
 
+    def get_fbxexport_options(self):
+        ''' Set the good options for fbx export to Sketchfab '''
+        # Get the fbx export plugin
+        fbxplugin = plugins.FindPlugin(1026370, c4d.PLUGINTYPE_SCENESAVER)
+        if not fbxplugin:
+            return
+        # Access the plugin options
+        reply = {}
+        if fbxplugin.Message(c4d.MSG_RETRIEVEPRIVATEDATA, reply):
+            if 'imexporter' in reply:
+                return reply['imexporter']
+            else:
+                return None
```

Comment:
the pythonic way to do this is: `reply.get('imexporter')` (by default `get` will return `None` if the key is not in the dict)